src/router/Datasets/Datasets.vue

```diff
@@ -125,15 +125,34 @@
           }
         },
         immediate: true
+      },
+      /**
+       * Not sure if this ever is the case but if the datasetID changes,
+       * we should start listening to new pusher channel
+       */
+      datasetId: function() {
+        this.$pusher.unsubscribe(this.pusherChannelName)
+        this.pusherChannelName = this.datasetId.replace("N:dataset:","dataset-")
+        console.log("Updating pusher in watcher: " + this.pusherChannelName)
```

Comment:
I noticed there are a number of console.logs here in the code - I have a note to myself to stage a PR to remove these prior to the site going live - but don't want to remove ones that you are still using for testing purposes - when does it make sense for us to remove logs? I typically don't let console.logs go out to production.